qiskit/transpiler/passmanager.py

```diff
@@ -46,6 +46,28 @@ def __init__(self, passes: Union[BasePass, List[BasePass]] = None, max_iteration
         self.max_iteration = max_iteration
         self.property_set = None
 
+    def __repr__(self):
+        if self.property_set is None or len(self.property_set) == 0:
+            nprops = 0
+            properties = "0 properties"
+        else:
+            prop_keys = list(self.property_set.keys())
+            nprops = len(prop_keys)
+            if nprops > 3:
+                _plist = "','".join(prop_keys[0:3]) + "',..."
+            else:
+                _plist = "','".join(prop_keys) + "'"
+            properties = f"{nprops} properties ('{_plist},)"
```

Comment:
I think let's leave out `property_set` from `PassManager` - it's an important component, sure, but I think it's really a subcomponent that people should inspect manually, since it only becomes set after the pass manager has been used, and the property set is really its own thing.